### PR TITLE
Fix deleting all artboards without crashing

### PR DESCRIPTION
**Pull request by 0HyperCube**
Delete all artboards now doesn't crash.

### CODE REVIEW

**Comment by Keavon** [comment]
The functionality I had in mind was removing the artboards but not the artwork contained within. Basically transplanting all the artwork to an infinite canvas. Would that be a change you can make, please?

File: editor/src/messages/portfolio/document/node_graph/graph_operation_message_handler.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 use super::{resolve_document_node_type, VectorDataModification};
 use crate::messages::prelude::*;
+use crate::messages::tool::common_functionality::graph_modification_utils::is_artboard;
```
Comment (Keavon) [inline editor/src/messages/portfolio/document/node_graph/graph_operation_message_handler.rs:3]: This unused import can be removed.

**Comment by 0HyperCube** [comment]
> The functionality I had in mind was removing the artboards but not the artwork contained within. Basically transplanting all the artwork to an infinite canvas. Would that be a change you can make, please?

If you have placed artwork as a child of an artboard, it will be deleted when the parent artboard is deleted. However you could move the arwork to outside of the artboard if you wish to avoid this.

**Comment by 0HyperCube** [comment]
If you decide you want deleting a group or artboard not to delete child layers, then you are welcome to open a separate issue for that and then implement it when you have time.

**Comment by Keavon** [comment]
> If you decide you want deleting a group or artboard not to delete child layers, then you are welcome to open a separate issue for that and then implement it when you have time.

Sounds good, I'll merge this now as-is then!

**Review by Keavon — APPROVED** [review]

**Pull request merged**